tests/test_logging/expected.txt

```diff
@@ -1,4 +1,10 @@
-           INFO                                                                                                                                                                                                                         scene_file_writer.py:   
-                    File ready at                                                                                                                                                                                                                               
+           INFO     Read configuration files: ['C:\\Users\\User\\OneDrive\\Bureau\\Programmation\\PYTHON\\MANIM-DEV\\manim\\manim\\default.cfg', 'C:\\Users\\User\\AppData\\Roaming\\Manim\\manim.cfg',                                            config.py:   
+                    'C:\\Users\\User\\OneDrive\\Bureau\\Programmation\\PYTHON\\MANIM-DEV\\manim\\tests\\tests_data\\manim.cfg']                                                                                                                                 
+           DEBUG    Animation   : Partial movie file written in                                                                                                                                                                        scene_file_writer.py:    
+                                                                                                                                                                                                                                                                
+           DEBUG    Animation   : Partial movie file written in                                                                                                                                                                        scene_file_writer.py:    
                                                                                                                                                                                                                                                                 
+           DEBUG    Animation   : Partial movie file written in                                                                                                                                                                        scene_file_writer.py:    
+                                                                                                                                                                                                                                                                
+           INFO                                                                                                                                                                                                                        scene_file_writer.py:    
```

Comment:
Is it absolutely guaranteed that there will only be these many debug messages?

Reply:
yep.
hashing won't be triggered as the media_temp direcyoty (where partial movies file are contained) is cleaned after the test.

Reply:
Cool. So, if I haven't forgotten anything, and if the `expected.txt` has been updated properly, the "logging" test must not fail.

I am not sure about the subcommand one. Does it work?